src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/EndpointSecurityTest.kt

```diff
@@ -0,0 +1,63 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource
+
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.fail
+import org.springframework.context.annotation.ClassPathScanningCandidateComponentProvider
+import org.springframework.core.type.filter.AnnotationTypeFilter
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.RestController
+import java.lang.reflect.AnnotatedElement
+import java.lang.reflect.Method
+
+class EndpointSecurityTest {
```

Comment:
A test that will fail if we add an endpoint and forget to add the appropriate role checks or [excluding annotation](https://github.com/ministryofjustice/create-and-vary-a-licence-api/pull/308/files#diff-70a75ff545f9111231777febe63f1dfbfcacd31f91510d73b5a328fade47bf0f)